include/mbedtls/compat-1.3.h

```diff
@@ -1446,7 +1446,6 @@
 #define SSL_LEGACY_RENEGOTIATION MBEDTLS_SSL_LEGACY_RENEGOTIATION
 #define SSL_MAC_ADD MBEDTLS_SSL_MAC_ADD
 #define SSL_MAJOR_VERSION_3 MBEDTLS_SSL_MAJOR_VERSION_3
-#define SSL_MAX_CONTENT_LEN MBEDTLS_SSL_MAX_CONTENT_LEN
```

Comment:
Please revert this change. This is a major change affecting almost all existing users